pages/admin/permit-holders.tsx

```diff
@@ -141,64 +141,55 @@ function renderAddress({ value }: AddressProps) {
   );
 }
 
-type UserStatusProps = {
+type UserStatusBadgeProps = {
   value: 'active' | 'inactive';
 };
 
-function renderUserStatusBadge({ value }: UserStatusProps) {
+// User status cell in table
+function renderUserStatusBadge({ value }: UserStatusBadgeProps) {
   return (
     <Wrap>
-      <Badge variant={value}>{value.toUpperCase()}</Badge>
+      <Badge variant={value}>{value?.toUpperCase()}</Badge>
```

Comment:
Same thing here with the `undefined` issue.

Also, can just throw this render function in the cells configuration.